lexicons/app/bsky/graph/assertion.json

```diff
@@ -1,22 +1,27 @@
 {
   "lexicon": 1,
   "id": "app.bsky.graph.assertion",
-  "type": "record",
-  "key": "tid",
-  "record": {
-    "type": "object",
-    "required": ["assertion", "subject", "createdAt"],
-    "properties": {
-      "assertion": {"type": "string"},
-      "subject": {
+  "defs": {
+    "main": {
+      "type": "record",
+      "key": "tid",
+      "record": {
         "type": "object",
-        "required": ["did", "declarationCid"],
+        "required": ["assertion", "subject", "createdAt"],
         "properties": {
-          "did": {"type": "string"},
-          "declarationCid": {"type": "string"}
+          "assertion": {"type": "string"},
```

Comment:
Do we want to drop some `knownValues` here using the assertion tokens that were setup?

Reply:
That would make sense. I'd suggest we have a separate PR after this where we do a refactor of the lexicons, to reuse some definitions and fix bits like that.